backend/apps/volontulo/factories.py

```diff
@@ -198,8 +201,9 @@ def volunteers(self, create, extracted):
     benefits = factory.Faker("paragraph")
     location = factory.Faker("address", locale="pl_PL")
     title = factory.Faker("text", max_nb_chars=150)
+#    import pdb; pdb.set_trace() - zrobić snippet // life templates
     started_at = factory.fuzzy.FuzzyDateTime(_start_date, _end_date)
-    finished_at = factory.fuzzy.FuzzyDateTime(_start_date, _end_date)
+    finished_at = factory.fuzzy.FuzzyDateTime(__start_date, __end_date)
```

Comment:
Beside @jacol12345 comments we have here one major problem - in that setting offer will never have `finished_at` in past, it will always be between `now + 101 days` and `now + 200 days` - we need to make it possible to generate offers in the past.

Moreover if You need to create these `__start_data` and `__end_date` variables (I would say, that You don't), make them more descriptive. I would even prefer to create in `class` only `_now = timezone.now()` and then:
```
started_at = factory.fuzzy.FuzzyDateTime(_now - datetime.timedelta(...), _now + datetime.timedelta(...))
finished_at = factory.fuzzy.FuzzyDateTime(_now - datetime.timedelta(...), _now + datetime.timedelta(...))
```